services/fitter/fitter.py

```diff
@@ -11,28 +11,29 @@
 
 sys.path.append(".")
 from services.plotter.models import Configurations
+from services.plotter.plotter import graph_builder, get_data
 
 
-def get_data(file_path):
-    df = pd.read_csv(file_path, sep=",", index_col=0)
-    return df
+# def get_data(file_path):
```

Comment:
Instead of commenting, just remove please.

Reply:
Done